piccolo/apps/playground/commands/run.py

```diff
@@ -288,14 +289,15 @@ def run(
     password: str = "piccolo",
     database: str = "piccolo_playground",
     host: str = "localhost",
-    port: int = 5432,
+    port: Optional[int] = None,
     ipython_profile: bool = False,
 ):
     """
     Creates a test database to play with.
 
     :param engine:
-        Which database engine to use - options are sqlite or postgres
+        Which database engine to use - options are sqlite, postgres or
+        cockroach
     :param user:
         Postgres user
```

Comment:
We need to update these params - e.g. `Postgres/CockroachDB user`.

Or `Database user (ignored for SQLite)`?